src/main/java/github/scarsz/discordsrv/DiscordSRV.java

```diff
@@ -721,7 +726,12 @@ private List<InetAddress> lookupPublic(String host) throws UnknownHostException
         dispatcher.setMaxRequestsPerHost(20); // most requests are to discord.com
         ConnectionPool connectionPool = new ConnectionPool(5, 10, TimeUnit.SECONDS);
 
-        OkHttpClient httpClient = new OkHttpClient.Builder()
+        String proxyHost = config.getString("ProxyHost").trim();
+        String proxyPort = config.getInt("ProxyPort").trim();
```

Comment:
```suggestion
        int proxyPort = config.getInt("ProxyPort");
```
I'm honestly surprised `trim()` worked here on `getInt()`.